metpy/plots/tests/test_skewt.py

```diff
@@ -290,11 +290,24 @@ def test_hodograph_plot_arbitrary_layer():
     v = np.arange(-5, -65, -5) * units('knot')
     speed = np.sqrt(u ** 2 + v ** 2)
     colors = ['red', 'green', 'blue']
-    levels = [0, 10, 20, 30] * units('knot')
+    levels = [0, 10, 20, 30] * units('kgit stnot')
```

Comment:
🤣 

How does this manage to pass!?! You should probably investigate.

Reply:
CI did fail... apparently part of `git status` made it in there... LOL!